cypress/features/regression/experimental/dateInputValidations.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Date Input validations component
+    I want to change Date Input validations component properties
+
+    Background: Open Date Input validations component page
+        Given I open "Experimental Date Input" component page validations
+
+    @positive
+    @validations
+    Scenario Outline: Verify the <state> validation of Date Input component
+        When I type specific date "<date>"
+            And I click onto specific day "<specificDay>" via DayPicker for validation component
```

Comment:
to much spaces / tabs before `And` we should use VSC formatting

Reply:
fixed